src/core/segment_sinks/inventory/segment_inventory.ts

```diff
@@ -221,6 +221,14 @@ export default class SegmentInventory {
     /** Type of buffer considered, used for logs */
     const bufferType: string | undefined = thisSegment?.infos.adaptation.type;
 
+    if (log.hasLevel("DEBUG")) {
+      const prettyPrintedRanges = ranges.map((r) => `${r.start}-${r.end}`).join(",");
+      log.debug(
+        `SI: synchronizing ${bufferType ?? "unknown"} buffered ranges:`,
+        prettyPrintedRanges,
+      );
+    }
```

Comment:
I added a log so it becomes easier to determine if a `SegmentInventory`'s synchronization result (logged at the end) do seem to correspond to its `SourceBuffer`'s `buffered` property at that time (logged here)